plugins/gcov/README.md

```diff
@@ -7,9 +7,10 @@ Plugin for integrating GNU GCov code coverage tool into Ceedling projects.
 Currently only designed for the gcov command (like LCOV for example). In the
 future we could configure this to work with other code coverage tools.
 
-This plugin currently uses [`gcovr`](https://www.gcovr.com/) as a utility to
-generate HTML, XML, JSON, or Text reports. The normal gcov plugin _must_ be run
-first for these reports to generate.
+This plugin currently uses [gcovr](https://www.gcovr.com/) and / or
+[Report Generator](https://danielpalme.github.io/ReportGenerator/)
```

Comment:
I think the name of tool is the ReportGenerator without space.

Reply:
Changed "Report Generator" to "ReportGenerator" in the README, comments, and strings.